torchaudio/datasets/utils.py

```diff
@@ -52,67 +52,6 @@ def stream_url(url: str,
             pbar.update(len(chunk))
 
 
-def download_url(url: str,
```

Comment:
We cannot delete the function until the next release. The `deprecated` decorator has to be put on this `download_url`.